test_check_bagged_scores fails with pandas 0.25+

Using the latest version of dependencies used by default in [environment.yml](https://github.com/paris-saclay-cds/ramp-board/blob/master/environment.yml) (at present pandas 0.25.2) `test_check_bagged_scores` fails with,
```
______________________________________________________ test_check_bagged_scores ________________________________
_______________________

session_scope_module = <sqlalchemy.orm.session.Session object at 0x7fe3898b91d0>

    def test_check_bagged_scores(session_scope_module):
        # check both set_bagged_scores and get_bagged_scores
        submission_id = 1
        path_results = os.path.join(HERE, 'data', 'iris_predictions')
        set_bagged_scores(session_scope_module, submission_id, path_results)
        scores = get_bagged_scores(session_scope_module, submission_id)
        multi_index = pd.MultiIndex(levels=[['test', 'valid'], [0, 1]],
                                    labels=[[0, 0, 1, 1], [0, 1, 0, 1]],
                                    names=['step', 'n_bag'])
        expected_df = pd.DataFrame(
            {'acc': [0.70833333333, 0.70833333333, 0.65, 0.6486486486486],
             'error': [0.29166666667, 0.29166666667, 0.35, 0.35135135135],
             'nll': [0.80029268745, 0.66183018275, 0.52166532641, 0.58510855181],
             'f1_70': [0.66666666667, 0.33333333333, 0.33333333333,
                       0.33333333333]},
            index=multi_index
        )
        expected_df.columns = expected_df.columns.rename('scores')
>       assert_frame_equal(scores, expected_df, check_less_precise=True)
ramp_database/tools/tests/test_submission.py:401:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
_ _ _ _ _ _ _ _ _ _ _ _
pandas/_libs/testing.pyx:65: in pandas._libs.testing.assert_almost_equal
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
_ _ _ _ _ _ _ _ _ _ _ _

>   ???
E   AssertionError: MultiIndex level [0] are different
E
E   MultiIndex level [0] values are different (100.0 %)
E   [left]:  Index(['valid', 'valid', 'test', 'test'], dtype='object', name='step')
E   [right]: Index(['test', 'test', 'valid', 'valid'], dtype='object', name='step')

pandas/_libs/testing.pyx:178: AssertionError
```
the same test passes when downgrading to pandas 0.24.

There is probably no need to support a broad range of dependencies and pinning them to some specific versions should be enough (related to https://github.com/paris-saclay-cds/ramp-board/issues/185).

In any case, supported versions should have an explicit upper bound as otherwise tests will break when this package is installed in the future.

--- a/ramp-database/ramp_database/tools/submission.py
+++ b/ramp-database/ramp_database/tools/submission.py
@@ -469,7 +469,7 @@ def get_bagged_scores(session, submission_id):
     """
     submission = select_submission_by_id(session, submission_id)
     bagged_scores = {}
-    for step in ('valid', 'test'):
+    for step in ('test', 'valid'):
         score_dict = {}
         for score in submission.scores:
             score_all_bags = getattr(score, '{}_score_cv_bags'.format(step))

